Cargo.toml

```diff
@@ -67,17 +67,17 @@ chrono = { version = "0.4", optional = true }
 # Set default-features = false because async not working in wasm right now
 flatgeobuf = { version = "4.2.0", optional = true, default-features = false }
 futures = { version = "0.3", optional = true }
-gdal = { version = "0.16", optional = true }
+gdal = { version = "0.17", optional = true }
 geo = "0.28"
 geo-index = "0.1.1"
 geos = { version = "9.0", features = ["v3_11_0", "geo"], optional = true }
 geozero = { version = "0.13", features = ["with-wkb"], optional = true }
 half = { version = "2.4.1", optional = true }
-http-range-client = { version = "0.7.2", optional = true }
+http-range-client = { version = "0.8", optional = true }
 indexmap = { version = "2", optional = true }
 lexical-core = { version = "0.8.5", optional = true }
 num_enum = "0.7"
-object_store = { version = "0.10.0", optional = true }
+object_store = { version = "0.10", optional = true }
```

Comment:
If you bump the version here, you also have to bump the version in other libraries at the same time. That means that the Python and JS libraries have to be updated to 0.11 as well, and that other dependencies that use object store need to use 0.11 as well. 

So because `parquet` 52 depends on object-store ^0.10, we realistically won't be able to update to object store 0.11 until parquet 53 is released.

Reply:
Yuuuup CI found that for me :-P ... I've backed that one off.